internal/aws/awsutil/conn.go

```diff
@@ -16,18 +16,22 @@
 package awsutil // import "github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/awsutil"
 
 import (
+	"context"
 	"crypto/tls"
 	"errors"
 	"net/http"
 	"net/url"
 	"os"
 	"time"
 
+	awsSDKV2 "github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/retry"
+	awsConfig "github.com/aws/aws-sdk-go-v2/config"
```

Comment:
Why do we need to rename this? I don't see another import here that would conflict

Reply:
Does not need to removed.